cypress/e2e/modal-overlays.spec.js

```diff
@@ -0,0 +1,29 @@
+/// <reference types="cypress" />
+
+describe('Modal&Overlays', ()=>{
+    
+
+    it('Hover the element without clicking it', ()=> {
+        cy.visit('/pages/modal-overlays/popover').wait(3000)
+        cy.get('nb-card-body button').contains('Right').trigger("mouseenter").then(el=>{
+            cy.get('nb-popover')
+            .contains("Hello, how are you today?")
+        })
+    })
+
+    it('Enter, Submit and Assert', ()=>{
+
+        cy.visit('/pages/modal-overlays/dialog').wait(3000)
+        cy.get('.result-from-dialog > .appearance-filled').click().get('.ng-star-inserted > nb-card > nb-card-body > .size-medium')
```

Comment:
```suggestion
        cy.get('.result-from-dialog > .appearance-filled').click()
        cy.get('.ng-star-inserted > nb-card > nb-card-body > .size-medium')
```
Кліком ти завершила дію, далі заново виклич cy.get і виконай дію